lib/upgrade/etcd/conversionv1v3/rule.go

```diff
@@ -77,8 +77,8 @@ func ruleAPIToBackend(ar apiv1.Rule) model.Rule {
 		NotICMPType: notICMPType,
 
 		SrcTag:      ar.Source.Tag,
-		SrcNet:      ar.Source.Net,
-		SrcNets:     ar.Source.Nets,
+		SrcNet:      normalizeIPNet(ar.Source.Net),
```

Comment:
If we weren't normalizing the CIDRs here before then there is the possibility that the backend model written to the datastore might be incorrect.  If that is the case then I think we'll also need to normalize these IPNets when converting from the backend to the v3 API.

Reply:
Also, there are no tests for this change.

Reply:
this code is only for the upgrade converters, so I don't see how anyone could have the wrong data in their backend yet. I guess I should change it for v1 bacendToAPI

Reply:
updated the conversion tests (rules_def.go) to expect normalized nets